charts/tekton-apps/Chart.yaml

```diff
@@ -1230,6 +1180,10 @@ description: |
                     repository: xxx-wordpress
                     pipeline: wordpress-build-pipeline
                     applicationURL: https://xxx.site.url
+                    argocd:
+                      source:
+                      targetRevision: 16.1.14
+                      namespace: wordpress
```

Comment:
rename this prop as `destinationNamespace` so it is semantic.

Reply:
Done